static/docs/commands-reference/destroy.md

```diff
@@ -39,15 +39,57 @@ $ echo foo > foo
 $ dvc add foo
 $ ls -a
 
-.dvc .git code.py foo foo.dvc
+.dvc .git foo foo.dvc
 
 $ dvc destroy
 
-This will destroy all information about your pipelines as well as cache in .dvc/cache.
-Are you sure you want to continue?
+This will destroy all information about your pipelines, all data files, as well
+as cache in .dvc/cache.
+Are you sure you want to continue? [y/n]
 yes
 
 $ ls -a
 
-.git code.py foo
+.git .gitignore
```

Comment:
this should not be the case, unless there is some serious bug

Reply:
```

ryokugyu@ub:~$ mkdir destroy_test
ryokugyu@ub:~$ cd destroy_test/
ryokugyu@ub:~/destroy_test$ git init
Initialized empty Git repository in /home/ryokugyu/destroy_test/.git/
ryokugyu@ub:~/destroy_test$ dvc init
Adding '.dvc/state' to '.dvc/.gitignore'.
Adding '.dvc/lock' to '.dvc/.gitignore'.
Adding '.dvc/config.local' to '.dvc/.gitignore'.
Adding '.dvc/updater' to '.dvc/.gitignore'.
Adding '.dvc/updater.lock' to '.dvc/.gitignore'.
Adding '.dvc/state-journal' to '.dvc/.gitignore'.
Adding '.dvc/state-wal' to '.dvc/.gitignore'.
Adding '.dvc/cache' to '.dvc/.gitignore'.

You can now commit the changes to git.

+---------------------------------------------------------------------+
|                                                                     |
|        DVC has enabled anonymous aggregate usage analytics.         |
|     Read the analytics documentation (and how to opt-out) here:     |
|              https://dvc.org/doc/user-guide/analytics               |
|                                                                     |
+---------------------------------------------------------------------+

What's next?
------------
- Check out the documentation: https://dvc.org/doc
- Get help and share ideas: https://dvc.org/chat
- Star us on GitHub: https://github.com/iterative/dvc
ryokugyu@ub:~/destroy_test$ echo foo > foo
ryokugyu@ub:~/destroy_test$ ls -a
.  ..  .dvc  foo  .git
ryokugyu@ub:~/destroy_test$ dvc add foo
Adding 'foo' to '.gitignore'.
Saving 'foo' to '.dvc/cache/d3/b07384d113edec49eaa6238ad5ff00'.
Saving information to 'foo.dvc'.

To track the changes with git run:

	git add .gitignore foo.dvc
ryokugyu@ub:~/destroy_test$ dvc destroy 
This will destroy all information about your pipelines, all data files, as well as cache in .dvc/cache.
Are you sure you want to continue? [y/n] y
ryokugyu@ub:~/destroy_test$ ls -a
.  ..  .git  .gitignore
```